Library/Formula/unbound.rb

```diff
@@ -19,13 +19,10 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-libevent=#{Formula["libevent"].opt_prefix}",
                           "--with-ssl=#{Formula["openssl"].opt_prefix}"
+    inreplace "doc/example.conf", 'username: "unbound"', "username: \"#{ENV["USER"]}\""
```

Comment:
How about something like this: we do a build-time replacement with a sentinel value, so we can tell if it's a "default" config file that this installation run just dropped, and then conditionally replace it.

```
def install
  ...
    inreplace "doc/example.conf", 'username: "unbound"', 'username: "default-unbound-user"'
  ...
end

def post_install
  if File.read(etc/"unbound/unbound.conf").include?('username: "default-unbound-user"')
    inreplace etc/"unbound/unbound.conf", 'username: "default-unbound-user"', "username: \"#{ENV["USER"]}\""
  end
end

Or maybe the test should be to check whether that `etc/unbound/unbound.conf` file exists at `pre_install` time, and if so, then do the replacement in `post_install`. That seems more congruent with the "only modify files that this installation run has created" intent.